aries-bookkeeper/src/main/java/org/apache/aries/AbstractBookkeeperToy.java

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright (c) 2019 R.C
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.aries;
+
+public abstract class AbstractBookkeeperToy extends AbstractToy {
```

Comment:
can put some common used object into here, instead of new() each time in subclass

Reply:
Done